src/Evolution/Systems/RadiationTransport/M1Grey/Fluxes.hpp

```diff
@@ -0,0 +1,99 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"  // for item_type
+#include "DataStructures/DataBox/Prefixes.hpp"    // IWYU pragma: keep
+#include "DataStructures/Tensor/TypeAliases.hpp"  // IWYU pragma: keep
+#include "Evolution/Systems/GrMhd/ValenciaDivClean/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "Evolution/Systems/RadiationTransport/M1Grey/Tags.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"  // IWYU pragma: keep
+#include "Utilities/Gsl.hpp"   // for not_null
+#include "Utilities/TMPL.hpp"  // for EXPAND_PACK_LEFT_TO...
+
+// IWYU pragma: no_include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+
+// IWYU pragma: no_forward_declare Tags::Flux
+
+/// \cond
+class DataVector;
+/// \endcond
+
+// IWYU pragma: no_forward_declare Tensor
+
+namespace RadiationTransport {
+namespace M1Grey {
+
+/// Implementation of the M1 fluxes for individual neutrino species
+namespace detail {
+void compute_fluxes_impl(
+    gsl::not_null<tnsr::I<DataVector, 3, Frame::Inertial>*> tilde_e_flux,
+    gsl::not_null<tnsr::Ij<DataVector, 3, Frame::Inertial>*> tilde_s_flux,
+    const Scalar<DataVector>& tilde_e,
+    const tnsr::i<DataVector, 3, Frame::Inertial>& tilde_s,
+    const tnsr::II<DataVector, 3, Frame::Inertial>& tilde_p,
+    const Scalar<DataVector>& lapse,
+    const tnsr::I<DataVector, 3, Frame::Inertial>& shift,
+    const tnsr::ii<DataVector, 3, Frame::Inertial>& spatial_metric,
+    const tnsr::II<DataVector, 3, Frame::Inertial>&
+        inv_spatial_metric) noexcept;
+}  // namespace detail
+
+/*!
+ * \brief The fluxes of the conservative variables in the M1 scheme
+ *
+ * \f{align*}
+ * F^i({\tilde E}) = &~ \alpha \gamma^{ij} {\tilde S}_j - \beta^j {\tilde E} \\
+ * F^i({\tilde S}_j) = &~ \alpha {\tilde P}^{ik} \gamma_{kj} - \beta^i {\tilde
+ * S}_j \f}
+ *
+ * where the conserved variables \f${\tilde E}\f$, \f${\tilde S}_i\f$,
+ * are a generalized mass-energy density and momentum density.
+ * Furthermore, \f${\tilde P^{ij}\f$ is the pressure tensor density of the
```

Comment:
You have an opening `{` without a closing `}` so this line isn't rendering correctly.